frame/multisig/src/tests.rs

```diff
@@ -162,105 +159,14 @@ fn multisig_deposit_is_taken_and_returned() {
 			2,
 			vec![1, 3],
 			Some(now()),
-			OpaqueCall::from_encoded(data),
-			false,
+			call,
 			call_weight
 		));
 		assert_eq!(Balances::free_balance(1), 5);
 		assert_eq!(Balances::reserved_balance(1), 0);
 	});
 }
 
-#[test]
-fn multisig_deposit_is_taken_and_returned_with_call_storage() {
```

Comment:
IIRC, there is still a fixed deposit that is collected, which you brought back in this PR. 

Do we still have tests for this. Should we keep this test as `multisig_deposit_is_taken_and_returned`? or do we already have something like that?

Reply:
yeah, this is tested in another scenario afaik, I will double-check

Reply:
https://github.com/paritytech/substrate/blob/7bacd730ff42afe9d6c5039a51e74bd461dac5ef/frame/multisig/src/tests.rs#L137 duplicate test